data/i18n/ar/ar.toml

```diff
@@ -0,0 +1,568 @@
+# i18n strings for the English (main) site.
+# NOTE: Please keep the entries in alphabetical order when editing
+
+# For "and", see [conjunction_1]
+
+[auto_generated_edit_notice]
+other = "(صفحة مولدة آلياً)"
+
+[auto_generated_pageinfo]
+other = """<p>هذه الصفحة مولدة آلياً،</p>
+<p>فإذا أردت الإبلاغ عن مشكلة فيها، فالرجاء ذكر ذلك، لاحتمال وجود سبب المشكلة في ملف آخر من المشروع.</p>"""
+
+[caution]
+other = "احذر:"
+
+[cleanup_heading]
+other = "جاري الترتيب"
+
+[community_events_calendar]
+other = "تقويم الفعاليت"
+
+[community_forum_name]
+other = "المنتدى"
+
+[community_github_name]
+other = "GitHub"
+
+[community_slack_name]
+other = "Slack"
+
+[community_stack_overflow_name]
+other = "Stack Overflow"
+
+[community_twitter_name]
+other = "تويتر"
+
+[community_youtube_name]
+other = "يوتيوب"
+
+# Avoid using conjunction_1.
+# Must match the context in layouts/shortcodes/release-data.html
+# Appears on https://kubernetes.io/releases/
+# For example the "and" in "Complete 1.25 Schedule and Changelog"
+[conjunction_1]
+other = "و"
+
+[cve_id]
+other = "CVE ID"
+
+[cve_issue_url]
+other = "رابط القضية للCVE"
+
+[cve_summary]
+other = "ملخص القضية"
+
+[cve_table]
+other = "القائمة الرسمية للCVE الخاصة بكوبرنيتيز"
```

Comment:
```suggestion
other = "القائمة الرسمية السفإ الخاصة بكاوبرناتيز"
```

Reply:
Given the ongoing [discussion](https://cloud-native.slack.com/archives/C02UFT8V256/p1707830370792469) about this translation—I will resolve this comment until we decide what the appropriate thing to do here is.

Reply:
I don't recommend resolving things if you're unsure whether the commenter would see the feedback as addressed.

Reply:
That's a good point, thanks @sftim